tests/test_extension_amber.py

```diff
@@ -17,6 +17,12 @@ def test_empty_snapshot(snapshot):
     assert snapshot == ""
 
 
+def test_newline_control_characters(snapshot):
+    assert snapshot == "line 1\nline 2"
+    assert snapshot == "line 1\r\nline 2"
```

Comment:
To see the broken reporting, remove the `\r`. I don't have a fix for the reporter.

Reply:
Should add an issue to track this

Reply:
https://github.com/tophat/syrupy/issues/116